lib/cve.py

```diff
@@ -153,19 +230,26 @@ def create_cves_record(cves):
     {"releaseNotes": {"cves": []}}
     """
 
-    result = {"releaseNotes": {"cves": []}}
-    
+    result = {
+        "releaseNotes": {
+            "cves": [
+            ]
+        }
+    }
+
     if cves:
-    
+
+        log(cves)
```

Comment:
This log should either be removed or it needs a title/description so users know what is being dumped out here.

Reply:
fixed